### PR TITLE
TDP-4050 document recording/transcription deletion

### PR DESCRIPTION
**Pull request by rmello-daitan**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

File: voice/methods/recordings/deleteCallsCallIdRecordingsRecordingId.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Cre
 |:----------|:------------|
 | None      | None        |
 
+Note: After the deletion is requested and a `204` is returned, the recording will not be accessible anymore. However, the media of the specified recording is not deleted immediately. The deletion process can take from 24 to 48 hours after it was requested.
```
Comment (nashley) [inline voice/methods/recordings/deleteCallsCallIdRecordingsRecordingId.md]:
```suggestion
Note: After the deletion is requested and a `204` is returned, neither the recording metadata nor the actual media nor its transcription will be accessible anymore. However, the media of the specified recording is not deleted immediately. This deletion process, while transparent and irreversible, can take an additional 24 to 48 hours.
```

File: voice/methods/recordings/deleteCallsCallIdRecordingsRecordingIdMedia.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Cre
 |:----------|:------------|
 | None      | None        |
 
+Note: After the deletion is requested and a `204` is returned, the recording will not be accessible anymore. However, the media of the specified recording is not deleted immediately. The deletion process can take from 24 to 48 hours after it was requested.
```
Comment (nashley) [inline voice/methods/recordings/deleteCallsCallIdRecordingsRecordingIdMedia.md]:
```suggestion
Note: After the deletion is requested and a `204` is returned, the media will not be accessible anymore. However, it is not deleted immediately. This deletion process, while transparent and irreversible, can take an additional 24 to 48 hours.
```

File: voice/methods/recordings/deleteCallsCallIdRecordingsRecordingIdTranscription.md [hunk 3]
```diff
@@ -20,6 +20,8 @@ Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Cre
 |:----------|:------------|
 | None      | None        |
 
+Note: After the deletion is requested and a `204` is returned, the transcription will not be accessible anymore. However, the transcription is not deleted immediately. The deletion process can take from 24 to 48 hours after it was requested.
```
Comment (nashley) [inline voice/methods/recordings/deleteCallsCallIdRecordingsRecordingIdTranscription.md]:
```suggestion
Note: After the deletion is requested and a `204` is returned, the transcription will not be accessible anymore. However, it is not deleted immediately. This deletion process, while transparent and irreversible, can take an additional 24 to 48 hours.
```

**Review by nashley — COMMENTED** [review]
These suggestions are a bit wordier, but I think it's good to be very specific.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4050-deletion-guarantees.s3-website-us-east-1.amazonaws.com/

**Comment by jchavez443** [comment]
The bucket did not deploy right.  You will need to use the below link for previews.

https://bw-dev-tdp-4050-deletion-guarantees.s3.amazonaws.com/index.html

**Comment by jchavez443** [comment]
The original link should work now as well, so if you make changes you can use any of the links.

**Review by n-miles — APPROVED** [review]

**Pull request merged**